R/notice.R

```diff
@@ -0,0 +1,34 @@
+
+# notice me senpai
+notice <- function(preamble,
```

Comment:
How about `caution_bullets()` or `caution_list()` as name? I think this should use `caution` as a prefix to tie it more closely to `caution()`.